resources/types/subnet.go

```diff
@@ -49,9 +50,13 @@ func NewSubnet(s *Subnet, resourceId string, subnet *resourcespb.SubnetArgs, oth
 }
 
 func (r *Subnet) Validate(ctx resources.MultyContext) (errs []validate.ValidationError) {
-	//if vn.Name contains not letters,numbers,_,- { return false }
-	//if vn.Name length? { return false }
-	//if vn.AvailbilityZone valid { return false }
+	//if vn.AvailbilityZone valid { return false }i
```

Comment:
since you're here, can you please remove this comment? it's no longer accurate as we removed av zones from subnets

Reply:
fixed;